packages/web-app-files/tests/__fixtures__/files.ts

```diff
@@ -0,0 +1,56 @@
+import { Resource } from 'web-client'
+
+export const files = [
```

Comment:
```suggestion
export const files: Resource[] = [
```

Can we do this instead of `as Resource[]`? That way we don't weaken the type checks